conf/app.routes

```diff
@@ -6,7 +6,7 @@ DELETE     /traders/:eori/records/:recordId         uk.gov.hmrc.tradergoodsprofi
 GET        /traders/:eori/records                   uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.GetRecordsController.getLocalRecords(eori: String, page: Option[Int], size: Option[Int])
 POST       /traders/:eori/records                   uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.CreateRecordController.createRecord(eori: String)
 GET        /traders/:eori/records/filter            uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.FilterRecordsController.filterLocalRecords(eori: String, searchTerm: Option[String], exactMatch: Option[Boolean], field: Option[String], page: Option[Int], size: Option[Int])
-HEAD       /traders/:eori/checkRecords              uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.CheckRecordsController.checkRecords(eori: String)
+HEAD       /traders/:eori/records-summary              uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.RecordsSummaryController.recordsSummary(eori: String)
```

Comment:
Making this change now will break the frontend, I'd recommend duplicating the endpoint and having them both call the same controller, then we can remove the old endpoint when the frontend is updated

Reply:
better idea